common/test/acceptance/tests/helpers.py

```diff
@@ -304,19 +306,20 @@ def reset_event_tracking(self):
         self.event_collection.drop()
         self.start_time = datetime.now()
 
-    def get_matching_events(self, event_type):
+    def get_matching_events(self, username, event_type):
         """
         Returns a cursor for the matching browser events.
         """
         return self.event_collection.find({
+            "username": username,
             "event_type": event_type,
             "time": {"$gt": self.start_time},
         })
 
-    def verify_events_of_type(self, event_type, expected_events, expected_referers=None):
+    def verify_events_of_type(self, username, event_type, expected_events, expected_referers=None):
         """Verify that the expected events of a given type were logged.
-
         Args:
+            username (str): The name of the authenticated user.
```

Comment:
Nit: The name of the user corresponding to the events that are being verified (or something similar).

It's just a little confusing why it is necessary to pass in the username...